pages/common/nano.md

```diff
@@ -1,28 +1,32 @@
 # nano
 
-> Simple, easy to use command-line text editor. An enhanced, free Pico clone.
+> Command-line text editor.
 > More information: <https://nano-editor.org>.
 
-- Open a new file in nano:
+- Start the editor:
 
 `nano`
 
-- Open a specific file:
+- Start the editor without loading startup configs:
 
-`nano {{path/to/file}}`
+`nano --ignorercfiles`
 
-- Open a specific file, positioning the cursor at the specified line and column:
+- Open specific files:
+
+`nano {{path/to/file1 path/to/file2 ...}}`
+
+- Open a file and position the cursor at a specific line and column:
```

Comment:
```suggestion
- Open a file and position the cursor at a specified line and column:
```

Suits better here

Reply:
As no rule exists whether `specific` or `specified` must be used it's up to me (in this case) what to prefer. I can't actually understand why it's better, what exactly better. :) That's why I mark this conversation as resolved.